federation/cmd/federation-apiserver/app/server.go

```diff
@@ -196,6 +197,8 @@ func Run(s *options.ServerRunOptions) error {
 		return err
 	}
 
+	routes.Version{}.Install(m.Mux, m.HandlerContainer)
```

Comment:
I assume you meant kubectl there and not kubelet?
Can you point to ldflags code you are talking about?
[base.go](https://github.com/kubernetes/kubernetes/blob/f2ddd60eb9e7e9e29f7a105a9a8fa020042e8e52/pkg/version/) just has strings